__tests__/app.test.js

```diff
@@ -8,62 +8,110 @@ beforeAll(() => seed(testData));
-            expect(res.body.msg).toBe('article not found');
-        })
-    });
-    test('/api/articles/:article_id 400: responds with a 400 error if article is not a number', () => {
-        return request(app)
-        .get('/api/articles/not-a-number')
-        .expect(400)
-        .then((res) => {
-            expect(res.body.msg).toBe('bad request');
-        })
     })
+    describe('GET /api/articles/:article_id', () => {
+        test('/api/articles/:article_id 200: responds with valid article object', () => {
+            return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then((res) => {
+                expect(res.body.article.article_id).toBe(1);
+                expect(res.body.article.title).toBe('Living in the shadow of a great man');
+                expect(res.body.article.topic).toBe('mitch');
+                expect(res.body.article.author).toBe('butter_bridge');
+                expect(res.body.article.body).toBe('I find this existence challenging');
+                expect(res.body.article.created_at).toBe('2020-07-09T20:11:00.000Z');
+                expect(res.body.article.votes).toBe(100);
+                expect(res.body.article.article_img_url).toBe('https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700');
+            });
+        });
+        test('/api/articles/:article_id 404: responds with a 404 error if article does not exist', () => {
+            return request(app)
+            .get('/api/articles/1000')
+            .expect(404)
+            .then((res) => {
+                expect(res.body.msg).toBe('article not found');
+            })
+        });
+        test('/api/articles/:article_id 400: responds with a 400 error if article is not a number', () => {
+            return request(app)
+            .get('/api/articles/not-a-number')
+            .expect(400)
+            .then((res) => {
+                expect(res.body.msg).toBe('bad request');
+            })
+        });
+    })
+    describe('GET /api/articles/article_id/comments', () => {
+        test('/api/articles/:article_id/comments 200: responds with a list of all comments for an article', () => {
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then((res) => {
+                expect(res.body.comments.length).toBeGreaterThanOrEqual(11);
+                expect(res.body.comments).toBeSorted('created_at', 'desc')
+                res.body.comments.forEach((comment) => {
+                    expect(typeof comment.comment_id).toBe('number');
+                    expect(typeof comment.article_id).toBe('number');
+                    expect(typeof comment.body).toBe('string');
+                    expect(typeof comment.votes).toBe('number');
+                    expect(typeof comment.author).toBe('string');
+                    expect(typeof comment.created_at).toBe('string');

```

Comment:
This works and its fine, but if it hasn't been mentioned already I'd start looking into expect objectcontaining and tomatchobject matchers for jest they allow for much more robust testing.